Firmware/code

```diff
@@ -0,0 +1,21 @@
+import dearpygui.dearpygui as dpg
```

Comment:
Probably should move this file into a new software folder instead of firmware since this isn't firmware.  I'd say give it its own folder named canalyzer_ui and name the file canalyzer_ui.py or something like that